### DISCUSSION TITLE
HCL vs. IBM in folder names

### DISCUSSION THREAD

**Discussion by marde16** [HCL-TECH-SOFTWARE/connections-automation#298] [Q&A]
Is there a reason for why some paths in the connections deployment contain HCL ...

https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/97eb462cd579cbb2f8accb63e22c7e81e5e3bcc4/roles/hcl/connections/vars/main.yml#L44

... and others IBM?

https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/97eb462cd579cbb2f8accb63e22c7e81e5e3bcc4/roles/hcl/connections/vars/main.yml#L38

https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/97eb462cd579cbb2f8accb63e22c7e81e5e3bcc4/roles/hcl/connections/vars/main.yml#L57

**Comment by sabrina-yee** [comment]
These are just standard paths we use in our internal deployments.  They can be overridden to align with the target environment.
